src/api/interfaces/pallet_balances.rs

```diff
@@ -0,0 +1,106 @@
+/*
```

Comment:
Hmm, `pallet_balances` is wrong actually, only the existential deposit belongs to the pallet balances. The rest is `pallet-transaction-payment`

Reply:
You're right, moved :)

Reply:
Yaay! 🥳